acceptance-tests/src/main/java/io/blueocean/ath/pages/blue/GithubCreationPage.java

```diff
@@ -26,18 +26,6 @@ public GithubCreationPage(WebDriver driver) {
         PageFactory.initElements(driver, this);
     }
 
-    @FindBy(css = "button.github-creation")
```

Comment:
I think we probably want to keep the strings themselves as final consts, but these aren't the kind of magic I like to purge anyway, as you could find them using Java resolution rather than relying on the implicit namespace where you're stringly-typed and using `@someNameYouCantResolve` inside selector strings.

I'll see mysef out.

Reply:
The annotation was just a wrapper around finding the element, the problem was the element was found when the page object was created, not when it the property was requested, so sometimes the element wasn't ready or had gone stale or whatever.

I'm not sold on constants for all the selectors, but its an easy change if desireable

Reply:
> the problem was the element was found when the page object was created, not when it the property was requested

Ew. Carry on 😀